layouts/shortcodes/all-contributors.html

```diff
@@ -0,0 +1,14 @@
+<div class="row justify-content-md-center">
+    {{ $data := getJSON "https://api.github.com/repos/Exodus-Privacy/website/contributors" }}
```

Comment:
```suggestion
    {{ $data := getJSON "https://api.github.com/orgs/Exodus-Privacy/members" }}
```

or
```suggestion
    {{ $data := getJSON "https://api.github.com/repos/Exodus-Privacy/exodus/contributors" }}
```

Reply:
Wait a min,

We want org members or inactive contributors?

+

Repo of website or app?

Reply:
> ```suggestion
>     {{ $data := getJSON "https://api.github.com/orgs/Exodus-Privacy/members" }}
> ```
> 
> or
> ```suggestion
>     {{ $data := getJSON "https://api.github.com/repos/Exodus-Privacy/exodus/contributors" }}
> ```

https://github.com/Exodus-Privacy/website/issues/85#issuecomment-1870970728